src/hotspot/share/gc/shared/referenceProcessor.cpp

```diff
@@ -225,29 +221,28 @@ ReferenceProcessorStats ReferenceProcessor::process_discovered_references(
                                 total_count(_discoveredPhantomRefs));
 
   {
-    RefProcTotalPhaseTimesTracker tt(RefPhase1, phase_times, this);
-    process_soft_ref_reconsider(is_alive, keep_alive, complete_gc,
-                                task_executor, phase_times);
+    RefProcTotalPhaseTimesTracker tt(RefPhase1, &phase_times, this);
```

Comment:
`RefProcTotalPhaseTimesTracker` doesn't really use `this`, right?

Reply:
Nice catch! Pre-existing, but I will clean it up.